InternshipBE/BL/Interfaces/IUserService.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Threading.Tasks;
+
+namespace BL.Interfaces
+{
+	public interface IUserService
+	{
+		Task<string> Authenticate(string login, string password);
```

Comment:
if the method is asynchronous, then at the end is added Async, 
rename to AuthenticateAsync